aict_tools/apply.py

```diff
@@ -106,9 +107,21 @@ def create_mask_h5py(
         name, (operator, value) = list(c.items())[0]
 
         before = mask.sum()
```

Comment:
While you on it, could you change all occurences of `mask.cum()` to `np.count_nonzero(mask)`

Reply:
Sure